package.json

```diff
@@ -88,5 +88,13 @@
       "src/Frontend/Cache/",
       "src/Frontend/Core/Js/jquery/jquery.js"
     ]
+  },
+  "volta": {
+    "node": "18.17.1"
```

Comment:
Waarom niet de laatste node versie? Dat is al v20 dacht ik?

Reply:
fixed 491695c970857ba513546729107dfceb18606cf7